projects/components/src/icon/icon.component.ts

```diff
@@ -9,21 +10,29 @@ import { IconSize } from './icon-size';
   styleUrls: ['./icon.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
-    <mat-icon
-      class="ht-icon"
-      [ngClass]="[this.size, this.borderType ? this.borderType : '']"
-      [ngStyle]="{
-        color: this.color ? this.color : '',
-        borderColor: this.borderType !== '${IconBorder.InsetBorder}' && this.borderColor ? this.borderColor : '',
-        background: this.borderType === '${IconBorder.InsetBorder}' && this.borderColor ? this.borderColor : '',
-        borderRadius: this.borderRadius ? this.borderRadius : ''
-      }"
-      [attr.aria-label]="this.labelToUse"
-      [htTooltip]="this.showTooltip ? this.labelToUse : undefined"
-      [fontSet]="this.fontSet"
-      [svgIcon]="this.svgIcon"
-      >{{ this.ligatureText }}</mat-icon
-    >
+    <ng-container>
```

Comment:
Why do we need an additional ng-container?

Reply:
removed.